src/Process/GlobeTileProcessing.js

```diff
@@ -1,9 +1,9 @@
 import * as THREE from 'three';
-import SchemeTile from '../Scene/SchemeTile';
 import MathExt from '../Core/Math/MathExtended';
+import SchemeTile from '../Core/Geographic/SchemeTile';
 import { UNIT, ellipsoidSizes } from '../Core/Geographic/Coordinates';
-import BoundingBox from '../Scene/BoundingBox';
-import { SIZE_TEXTURE_TILE } from '../Core/Commander/Providers/OGCWebServiceHelper';
+import { SIZE_TEXTURE_TILE } from '../Core/Scheduler/Providers/OGCWebServiceHelper';
+import Extent from '../Core/Geographic/Extent';
 
 const cV = new THREE.Vector3();
 let vhMagnitudeSquared;
```

Comment:
Yes this is a problem.
This is the same reason that makes the globe controls behave weirdly in the `multiglobe` example.